### ISSUE TITLE
Windows 7 32-bit : ZIP inconsistency

### ISSUE THREAD

**Issue by vikrantpogula** [mafintosh/electron-prebuilt#57]
I'm trying to run electron on a 32-bit windows 7 without any dependencies and it throws the following

```
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Windows\system32>npm install -g electron-prebuilt
C:\Users\Virtual\AppData\Roaming\npm\electron -> C:\Users\Virtual\AppData\Roamin
g\npm\node_modules\electron-prebuilt\cli.js

> electron-prebuilt@0.32.3 postinstall C:\Users\Virtual\AppData\Roaming\npm\node
_modules\electron-prebuilt
> node install.js

Downloading electron-v0.32.3-win32-ia32.zip
[============================================>] 100.0% of 39.71 MB (3.53 MB/s)
electron-prebuilt@0.32.3 C:\Users\Virtual\AppData\Roaming\npm\node_modules\elect
ron-prebuilt
├── extract-zip@1.1.0 (debug@0.7.4, async@0.9.0, mkdirp@0.5.0, yauzl@2.3.1, conc
at-stream@1.5.0)
└── electron-download@1.1.0 (path-exists@1.0.0, home-path@0.1.2, minimist@1.2.0,
 debug@2.2.0, mkdirp@0.5.1, mv@2.1.1, nugget@1.5.4)

C:\Windows\system32>electron
module.js:338
    throw err;
    ^

Error: Cannot find module 'C:\Users\Virtual\AppData\Roaming\npm\node_modules\ele
ctron-prebuilt\dist\resources\atom.asar\browser\lib\init.js'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Function.Module.runMain (module.js:471:10)
    at startup (node.js:139:18)
    at node.js:991:3

C:\Windows\system32>node --version
v4.0.0
```
its a complete fresh install and using node v4 as you can see above

**Comment by vikrantpogula** [comment]
Ok I seem to have figured out the issue by comparing with the prebuilt versions available here https://github.com/atom/electron/releases

It seems `npm install -g electron-prebuilt` is not including a default `.asar` archive (in this case `atom.asar`) in the resources folder. The following 2 folders are missing from the npm-install version and the prebuilt .zip

```
 - resources
 - locales
```

**Comment by mal** [comment]
Seems this is not exclusive to Windows, just got it on linux x64. Culprit looks to be the recent bump to the `extract-zip` modules, it doesn't seem to be extract directories. Filed an issue: https://github.com/maxogden/extract-zip/issues/10.

**extract-zip@1.1.0**
```diff
 $ npm install

 > electron-prebuilt@0.32.3 postinstall /home/mal/code/mal/photon/node_modules/electron-prebuilt
 > node install.js

 extract-zip@1.1.0 node_modules/extract-zip
 ├── debug@0.7.4
 ├── async@0.9.0
 ├── yauzl@2.3.1 (pend@1.2.0, fd-slicer@1.0.1)
 ├── mkdirp@0.5.0 (minimist@0.0.8)
 └── concat-stream@1.5.0 (inherits@2.0.1, typedarray@0.0.6, readable-stream@2.0.2)

 $ ll dist/
 total 93956
 -rw-r--r-- 1 mal mal  8743327 Sep 16 08:48 content_shell.pak
 -rwxr-xr-x 1 mal mal 58922768 Sep 16 08:48 electron
 -rw-r--r-- 1 mal mal 10206624 Sep 16 08:48 icudtl.dat
 -rw-r--r-- 1 mal mal   520152 Sep 16 08:48 libgcrypt.so.11
 -rwxr-xr-x 1 mal mal 16765152 Sep 16 08:48 libnode.so
 -rw-r--r-- 1 mal mal    31256 Sep 16 08:48 libnotify.so.4
 -rw-r--r-- 1 mal mal     1055 Sep 16 08:48 LICENSE
 -rw-r--r-- 1 mal mal   410849 Sep 16 08:48 natives_blob.bin
 -rw-r--r-- 1 mal mal   587068 Sep 16 08:48 snapshot_blob.bin
 -rw-r--r-- 1 mal mal        7 Sep 16 08:48 version
```

**extract-zip@1.0.3**
```diff
 $ npm install
 -
 > electron-prebuilt@0.32.3 postinstall /home/mal/code/mal/photon/node_modules/electron-prebuilt
 > node install.js

 extract-zip@1.0.3 node_modules/extract-zip
 ├── debug@0.7.4
 ├── minimist@0.1.0
 ├── async@0.9.0
 ├── yauzl@2.3.1 (pend@1.2.0, fd-slicer@1.0.1)
 ├── mkdirp@0.5.0 (minimist@0.0.8)
 ├── concat-stream@1.5.0 (inherits@2.0.1, typedarray@0.0.6, readable-stream@2.0.2)
 └── through2@0.6.3 (xtend@4.0.0, readable-stream@1.0.33)

 $ ll dist
 total 93964
 -rw-r--r-- 1 mal mal  8743327 Sep 16 08:47 content_shell.pak
 -rwxr-xr-x 1 mal mal 58922768 Sep 16 08:47 electron
 -rw-r--r-- 1 mal mal 10206624 Sep 16 08:47 icudtl.dat
 -rw-r--r-- 1 mal mal   520152 Sep 16 08:47 libgcrypt.so.11
 -rwxr-xr-x 1 mal mal 16765152 Sep 16 08:47 libnode.so
 -rw-r--r-- 1 mal mal    31256 Sep 16 08:47 libnotify.so.4
 -rw-r--r-- 1 mal mal     1055 Sep 16 08:47 LICENSE
+drwxrwxr-x 2 mal mal     4096 Sep 16 08:47 locales
 -rw-r--r-- 1 mal mal   410849 Sep 16 08:47 natives_blob.bin
+drwxrwxr-x 3 mal mal     4096 Sep 16 08:47 resources
 -rw-r--r-- 1 mal mal   587068 Sep 16 08:47 snapshot_blob.bin
 -rw-r--r-- 1 mal mal        7 Sep 16 08:47 version
```

**Comment by maxogden** [comment]
Thanks for the detailed bug reports, the issue has been fixed with extract-zip 1.1.1, so reinstalling electron-prebuilt should get the new working version
